lib/links/linkTypes/lib/smartArguments.js

```diff
@@ -7,21 +7,21 @@ export default new class {
     getIds(what, options) {
         if (Array.isArray(what)) {
             return _.map(what, (subWhat) => {
-                return this.getId(subWhat, options)
-            })
+                return this.getId(subWhat, options);
+            }).filter(id => _.isString(id));
         } else {
-            return [this.getId(what, options)];
+            return [this.getId(what, options)].filter(id => _.isString(id));
         }
 
         throw new Meteor.Error('invalid-type', `Unrecognized type: ${typeof what} for managing links`);
     }
 
     getId(what, options) {
-        if (typeof what === 'string') {
+        if (_.isString(what)) {
```

Comment:
Any reason why use underscore instead of native?

Reply:
I think I just followed the standard practice in the repo in 2019. 
I'll change it.

Reply:
Updated:
https://github.com/cult-of-coders/grapher/pull/403/commits/fb271e994ee473c6e448c7741cffb71da6f050e3